source/jormungandr/jormungandr/stat_manager.py

```diff
@@ -372,6 +397,15 @@ def fill_journey(self, stat_journey, resp_journey):
             if admin[2]:
                 stat_journey.last_pt_admin_name = admin[2]
 
+    def fill_tags(self, stat_journey, resp_journey):
+        result = set()
+        for tag in resp_journey.get("tags", []):
+            stat_tag = MAP_JOURNEY_TAG.get(tag, stat_pb2.JOURNEY_TAG_UNKNOWN)
+            if stat_tag == stat_pb2.JOURNEY_TAG_UNKNOWN:
+                logging.getLogger(__name__).warning("Stat tag not found for {} navitia tag.".format(tag))
+            result.add(stat_tag)
```

Comment:
I am not sure it is worthwhile to add the tag `JOURNEY_TAG_UNKNOWN` to the stats proto message :shrug:

Reply:
I think instead of empty value, better to use this one.
We better not communicate this information to any client and use only for internal analysis.